charts/ceph-csi-rbd/templates/nodeplugin-clusterrole.yaml

```diff
@@ -15,4 +15,19 @@ rules:
     resources: ["nodes"]
     verbs: ["get"]
 {{- end }}
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
```

Comment:
instead of creating two clusterroles i would like to do below

```yaml
{{- if .Values.rbac.create -}}
kind: ClusterRole
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: {{ include "ceph-csi-rbd.nodeplugin.fullname" . }}
  labels:
    app: {{ include "ceph-csi-rbd.name" . }}
    chart: {{ include "ceph-csi-rbd.chart" . }}
    component: {{ .Values.nodeplugin.name }}
    release: {{ .Release.Name }}
    heritage: {{ .Release.Service }}
rules:
{{- if .Values.topology.enabled }}
  - apiGroups: [""]
    resources: ["nodes"]
    verbs: ["get"]
{{- end }}
  - apiGroups: [""]
    resources: ["secrets"]
    verbs: ["get"]
{{- end -}}
```

Reply:
ok, done!